File: mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

for i in range(n):
    for j in range(n-1-i):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print("  ", end="")
    for j in range(i+1):
        print("#", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int height = get_int("Height: \n");
    //asks for height until it gets a proper one
    while(height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }

    //each i loop represents a level of the pyramid
    for(int i=1; i<=height; i++)
    {
        //spaces on left
        for(int j=0; j<height-i; j++)
        {
            printf(" ");
        }
        //hashtags on left
        for(int j=0; j<i; j++)
        {
            printf("#");
        }

        //middle
        printf("  ");

        //hashtags on right
        for(int j=0; j<i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}